.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: cpp
+
+os:
+- linux
+- osx
+
+sudo: true
+
+before_script:
+- sh install-deps.sh
```

Comment:
If you put the executable bit on this file, and a shebang, you don’t have to hardcode the interpreter to use.